packages/gafl-webapp-service/src/pages/renewals/identify/__tests__/identify.spec.js

```diff
@@ -87,7 +85,7 @@ describe('The easy renewal identification page', () => {
     expect(data.headers.location).toHaveValidPathFor(IDENTIFY.uri)
   })
 
-  it('redirects back to itself on posting an valid but not authenticated details', async () => {
+  it('redirects to licence not found on posting an valid but not authenticated details', async () => {
```

Comment:
Not a change you made but "an valid" is bugging me!
```suggestion
  it('redirects to licence not found on posting valid but not authenticated details', async () => {
```

Reply:
good spot!